### PR TITLE
Created KeyBoardWordFinderController.

### PR DESCRIPTION
**Pull request by xedisu**
Created index page.
Created keyboardWordFinderInputPage.html
Created keyboardWordFinderOutputPage.html
Created WordsDTO.java.
Created Application.java.

### CODE REVIEW

File: src/main/java/com/algorithmtron/Application.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.algorithmtron;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+
```
Comment (BogdanTanase) [inline src/main/java/com/algorithmtron/Application.java:6]: Do you need this space?

File: src/main/resources/templates/index.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset = "ISO-8859-1" />
+    <link href = "css/styles.css" rel = "stylesheet"/>
+    <title>Algorithmtron 2023</title>
+</head>
+<body>
+<h4>Welcome to Algo!</h4>
```
Comment (BogdanTanase) [inline src/main/resources/templates/index.html]: Perhaps a nicer greetings?

**Review by BogdanTanase — CHANGES_REQUESTED** [review]
Good job! Left some small comments :D

File: src/main/java/com/algorithmtron/Application.java (hunk 1 above)
Comment (xedisu) [reply src/main/java/com/algorithmtron/Application.java:6]: You're right, fixed!

**Review by BogdanTanase — APPROVED** [review]

**Pull request merged**